state-storage/lib/state-storage.ts

```diff
@@ -3,6 +3,20 @@ import {Logger} from "@sailplane/logger";
 
 const logger = new Logger('state-storage');
 
+interface StateStorageOptions {
+    /** If true, do not log values. */
+    quiet?: boolean;
+    /**
+     * If true, store as encrypted or decrypt on get. Uses account default KMS key.
+     * Implies quiet as well.
+     */
+    secure?: boolean;
+    /**
+     * If set, set and get the value as is, not JSON. (Only works for string values.)
+     */
+    isRaw?: boolean;
```

Comment:
nit: At first, I didn't like the name of this option.  If it's not raw, then what is it?  (JSON obviously.).   Then, I wondered why we need to expose this parameter at all.  If the incoming object is anything but a string, then `JSON.stringify` it.  Upon retrieval, if JSON.parse succeeds (in a `try` clause), return it.  If there's a failure parsing, return the raw (string) value.